firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-# Unreleased (21.3.1)
+# Unreleased (21.3.2)
+- [changed] Firestore previously required that every document read in a
```

Comment:
FWIW You could make this a feature if you think it is more than a bug fix. You would then bump the version to 21.4.

Reply:
Going point. I think it's worth flagging this as a `feature` and bumping the minor version.